### PR TITLE
Plamen5kov/respect shelljserrors

### PR DESCRIPTION
**Pull request by Plamen5kov**
This PR should allow shelljs errors to be fatal and interrupt the execution of any command if necessary. The current implementation has accounted for several places where, the errors are expected and are handled accordingly.
One more thing is the minimal project template support version is upped from `1.3.0` to `1.5.0`.

This PR should be merged after: https://github.com/telerik/mobile-cli-lib/pull/861

### CODE REVIEW

**Comment by Plamen5kov** [comment]
run CI

File: lib/services/project-service.ts [hunk 1]
```diff
@@ -140,7 +140,8 @@ export class ProjectService implements IProjectService {
 
 			shelljs.cp('-R', path.join(appSourcePath, "*"), appDestinationPath);
 			// Copy hidden files.
-			shelljs.cp('-R', path.join(appSourcePath, ".*"), appDestinationPath);
+			//TODO: plamen5kov: don't copy hidden files, until we find a reason to do it
+			// shelljs.cp('-R', path.join(appSourcePath, ".*"), appDestinationPath);
```
Comment (rosen-vladimirov) [inline lib/services/project-service.ts]: Strongly disagree here. Why you need a reason for already existing code? Either remove it or keep it in the way it works today. Also I can reverse the question - why wouldn't you copy all files from template? Hidden files could be part of the template, so... why not copying them? And last - on Windows they are not hidden at all.

**Comment by pkoleva** [comment]
run CI

**Comment by Plamen5kov** [comment]
run CI

**Pull request merged**